sdk/cognitiveservices/cognitiveservices-luis-runtime/package.json

```diff
@@ -4,8 +4,11 @@
   "description": "LUISRuntimeClient Library with typescript type definitions for node.js and browser.",
   "version": "3.0.1",
   "dependencies": {
-    "@azure/ms-rest-js": "^1.8.1",
-    "tslib": "^1.9.3"
+    "@azure/ms-rest-azure-js": "^2.0.1",
```

Comment:
Why is `@azure/ms-rest-azure-js` being added as a dependency?

Reply:
Because the tests will need it to authenticate as we have the tests and the sdk in one project.

Reply:
If only the tests need it, then it should be moved under `devDependencies`.
Otherwise, `@azure/ms-rest-azure-js` will get shipped as part of the luis library to the user